components/shared_code/src/shared_data_model/sources/dependency_track.py

```diff
@@ -138,6 +138,15 @@
                 EntityAttribute(name="Latest", key="is_latest", type=EntityAttributeType.BOOLEAN),
                 EntityAttribute(name="Last BOM import", type=EntityAttributeType.DATETIME),
                 EntityAttribute(name="Last BOM analysis", type=EntityAttributeType.DATETIME),
+                EntityAttribute(
+                    name="Up-to-date",
+                    color={
+                        "almost": Color.WARNING,
```

Comment:
Don't we use the term "near" instead of "almost", or is that only for parameter and not value?

Reply:
Yeah, I thought about that, but given that the attribute name is "Up-to-date", "near" doesn't make sense as an answer to the question "Is the the project's BOM up-to-date?". Maybe "nearly"?

By the way, I've also considered making the attribute name "Status", but that conflicts with the suppression status that all measurement entities have ("fixed", "false positive", etc.).

Reply:
What about using something like a label "age" with a values such as "new", "recent" and "old"? I think the trouble is mostly due to the term "up-to-dateness" which is uneasy, although it is an official noun.. But I'm also inclined to agree that all other options are worse than what you already chose

Reply:
Ha, I considered "Age" as well, but "new", "recent", and "old" don't really cover what is happening, namely that the age is compared to the metric target. If the target is 1 day, 2 days would already be listed as "old" which sounds weird. I'll change "almost " to "nearly" and leave it at that.